Makefile

```diff
@@ -58,7 +58,7 @@ java_kompiled:=$(java_dir)/test-kompiled/compiled.txt
 
 haskell_dir:=$(defn_dir)/haskell
 haskell_defn:=$(patsubst %, $(haskell_dir)/%, $(wasm_files))
-haskell_kompiled:=$(haskell_dir)/test-kompiled/kore.txt
+haskell_kompiled:=$(haskell_dir)/test-kompiled/compiled.txt
```

Comment:
The haskell backend does use `kore.txt` as the canonical source, not `compiled.txt`. Actually any of the backends using `compiled.txt` as the target file is probably not the best idea, since it's not actually used for anything.

Reply:
But there isn't much of a better choice for the Java backend.